### PR TITLE
Seperate deploy variables into production and testing

### PR DESCRIPTION
**Pull request by jamielennox**
Currently our deploy is targetted only at producing the production instance of BonnyCI. There are hardcoded references to BonnyCI urls and links between servers that will only work in production. Split these so that we can start to effectively test our deployment. 

Signed-off-by: Jamie Lennox <jamielennox@gmail.com>

### CODE REVIEW

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.com/check_github/BonnyCI/hoist/48/1484007399.18/ubuntu-hoist : FAILURE in 3m 38s

File: inventory/group_vars/multinode [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  - name: cicloud
+    cloud: cicloud
+    max-servers: 10
+    images:
+      - name: ubuntu-xenial
+        min-ram: 2048
+        diskimage: ubuntu-xenial
+        private-key: /var/lib/nodepool/.ssh/id_rsa
+
+nodepool_mysql_host: zuul.vagrant
```
Comment (ConsoleCatzirl) [inline inventory/group_vars/multinode]: This should be .multinode instead of .vagrant

**Review by ConsoleCatzirl — DISMISSED** [review]

File: inventory/group_vars/multinode (hunk 1 above)
Comment (jamielennox) [reply inventory/group_vars/multinode]: Yep, fixed.

**Comment by jamielennox** [comment]
For now the .vagrant or .multinode isn't actually going to be a problem because both will fail to resolve. The question is do we want to modify /etc/hosts to make them resolve or do we think we can use ansible ips?

**Comment by jamielennox** [comment]
recheck

**Comment by anne-bonny** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by jamielennox** [comment]
recheck

**Comment by rattboi** [comment]
recheck

**Review by omgjlk — APPROVED** [review]
There is a lot here, but I think it's all for the positive. Lets run with it!

**Comment by anne-bonny** [comment]
Starting gate_github jobs.

**Pull request merged**